src/dune_rules/install_rules.ml

```diff
@@ -520,7 +520,7 @@ end = struct
             let+ sub_systems =
               Lib.to_dune_lib lib
                 ~dir:(Path.build (lib_root lib))
-                ~modules ~foreign_objects
+                ~foreign_objects ~modules
```

Comment:
spurious change?

Reply:
yeah, I had removed it then added it back. Will revert